scripts/deployment/l2/bridges/gnosis/test_service_registry_l2_change_manager.js

```diff
@@ -0,0 +1,113 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```

Comment:
Script to change the manager for the ServiceRegistryL2 to swap ServiceManager with ServiceManagerToken on chiado.